src/sidebar/panel.html

```diff
@@ -4,6 +4,7 @@
   <meta charset="utf-8" />
   <title>Firefox Notes</title>
   <link href="vendor/quill.snow.css" rel="stylesheet">
+  <link rel="stylesheet" href="vendor/material.css">
```

Comment:
to keep the same code style as below put "href" first then "rel".  Also can we put it BEFORE the quill CSS so hopefully nothing in `material` overrides `quill`